about.css

```diff
@@ -0,0 +1,388 @@
+  font-size: 30px;
+}
+
+.plugDetHr {
+  margin-left: 150px;
+  margin-right: 150px;
+  color: red;
+}
+
+.valuePara {
+  text-align: center;
+  color: rgb(87, 88, 88);
+}
+
+.plugEsth {
+  font-size: 20px;
+}
+
+.plugEsDat {
+  border-bottom: 0.5px solid rgb(87, 88, 88);
+  border-top: 0.5px solid rgb(87, 88, 88);
+  font-size: 10px;
+  text-align: center;
+  margin-top: -30px;
+}
+
+.project {
+  display: flex;
+  flex-direction: column;
+  border-top: 1px solid rgb(87, 88, 88);
+  margin-top: 50px;
+}
+
+.projectH {
+  text-align: center;
+  font-size: 30px;
+}
+
+.projectHr {
+  color: red;
+  margin-left: 150px;
+  margin-right: 150px;
+  margin-top: -20px;
+}
+
+.projectPara {
+  text-align: center;
+  color: rgb(143, 147, 150);
+}
+
+.projectBack {
+  background-color: red;
+  opacity: 0.5;
+  height: 200px;
+  margin-top: -20px;
+  color: white;
+}
+
+.project1 {
+  background-image: url("img/project1.jpg");
+  background-size: cover;
+  text-align: center;
+  height: 200px;
+  width: 350px;
+}
+
+.project2 {
+  background-image: url("img/project2.jpg");
+  background-size: cover;
+  text-align: center;
+  width: 350px;
+  height: 200px;
+}
+
+.projectDet {
+  display: grid;
+  grid-template-columns: auto;
+  grid-gap: 20px;
+  justify-content: center;
+}
+
+footer {
+  margin-top: 100px;
+}
+
+.partners {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(58, 56, 56);
+  padding: 10px;
+}
+
+.partnersDiv {
+  text-align: center;
+  padding: 20px 0;
+  font-size: 30px;
+}
+
+.partOn {
+  display: flex;
+  justify-content: center;
+  gap: 80px;
+}
+
+.partTw {
+  justify-content: center;
+  gap: 80px;
+  display: flex;
+}
+
+.Div1 {
+  color: white;
+}
+
+.Div1Hr {
+  color: red;
+  margin-left: 150px;
+  margin-right: 150px;
+}
+
+.Div2 {
+  color: rgb(159, 162, 163);
+}
+
+.Div4 {
+  color: rgb(142, 145, 148);
+}
+
+.Div5 {
+  color: rgb(190, 195, 199);
+}
+
+.Div3 {
+  color: rgb(101, 103, 104);
+}
+
+.copyDiv {
+  display: flex;
+}
+
+.copyright {
+  display: flex;
+  margin-top: 90px;
+  justify-content: center;
+  align-items: center;
+  margin-left: 20px;
+  gap: 30px;
+}
+
+.plugicon {
+  height: 50px;
+}
+
+@media only screen and (min-width: 768px) {
+  .sideMback {
+    display: none;
+  }
+
+  .navSlide {
+    display: block;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .mediNbac {
+    background-color: black;
+    height: 30px;
+  }
+
+  .mediaNav {
+    display: flex;
+    margin-top: 3px;
+    float: right;
+    margin-right: 190px;
+  }
+
+  .mediaNav > li {
+    display: inline;
+    color: white;
+    margin-left: 10px;
+  }
+
+  .naVimg {
+    height: 18px;
+  }
+
+  .relNav {
+    display: flex;
+    background-color: white;
+    justify-content: center;
+    gap: 160px;
+    height: 60px;
+  }
+
+  .Menuli:hover {
+    border: 2px solid red;
+    color: red;
+  }
+
+  .Navle {
+    display: flex;
+    list-style-type: none;
+    margin-top: 0;
+  }
+
+  .NavRig {
+    float: left;
```

Comment:
- [ ] There are still some class names that start with uppercase letters please, make them lower case as the previous reviewer suggested.